src/modules/IscsiClientLib.rb

```diff
@@ -190,44 +190,49 @@ def hidePassword(orig)
       deep_copy(hidden)
     end
 
+    # Look for iSCSI boot firmware table (available only on special hardware/netcards)
+    #
+    # @return [String] stdout of command 'iscsiadm -m fw'
+    #
+    def getFirmwareInfo
+      bios_info = Convert.to_map(SCR.Execute(path(".target.bash_output"),
+                                             GetAdmCmd("-m fw")))
 
-    # get iBFT (available only on some special hardware)
+      fw_info = bios_info.fetch("stdout", "")
+
+      deep_copy(fw_info)
+    end
+
+    # Get iBFT (iSCSI boot firmware table) info
+    #
+    # @return [Hash] iBFT data converted to a hash (passwords are hidden)
+    #
     def getiBFT
       if @ibft == nil
         if !Arch.i386 && !Arch.x86_64
-          Builtins.y2milestone(
-            "Because architecture %1 is is different from x86, not using iBFT",
-            Arch.arch_short
-          )
+          log.info "Because architecture #{Arch.arch_short} is is different from x86, not using iBFT"
           return {}
```

Comment:
just note that there is problematic logic as result is cached in `@ibft` this workflow do not cache it, so it is checked everytime.